app/components/ui/GDocEmbed/GDocEmbed.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Iframe from 'react-iframe';
+import Footer from '../Footer/Footer';
+
+import styles from './GDocEmbed.scss';
+
+
+export default class GDocEmbed extends React.Component {
```

Comment:
Nit on naming (which admittedly are my personal style preferences): I'd prefer to spell out GDoc as GoogleDoc, since it's clearer when spelled out all the way, and it's not even an official abbreviation that Google uses in public materials.